packages/normalizr/src/entities/Entity.ts

```diff
@@ -258,25 +270,22 @@ First three members: ${JSON.stringify(input.slice(0, 3), null, 2)}`;
     // TODO: This creates unneeded memory pressure
     const instance = new (this as any)();
     let deleted = false;
-    const denormEntity = input;
 
     // note: iteration order must be stable
     Object.keys(this.schema).forEach(key => {
       const schema = this.schema[key];
-      const nextInput = this.hasDefined(input, key as any)
-        ? input[key]
-        : undefined;
+      const nextInput = key in input ? input[key] : undefined;
```

Comment:
`in` will also trigger for Object.prototype keys - the previous logic used `hasDefined` which implies it's checking hasOwnProperty.